### PR TITLE
Captain's winter coat has the same armor value's as his carapace

### PR DESCRIPTION
**Pull request by ItsTisme**
## About The Pull Request

Captains winter coat has the same armor values as his carapace

## Why It's Good For The Game

Cosmetic item that doesn't have less then half the stats. People who say that this is now superior to the normal armor because it gives minimal cold resistance id like to note that this makes the captains sprite bigger and there for easier to hit, so this balances out.

## Changelog
:cl:
tweak: caps winter coat now has the same armor value as his carapace

/:cl:

### CODE REVIEW

**Comment by zeskorion** [comment]
this covers arms, which the captain's carapace does not, adds more armor to his locker, and is as stupid as hos' locker full of 30 edgy trenchcoats

File: code/modules/clothing/suits/miscellaneous.dm [hunk 1]
```diff
@@ -557,9 +557,10 @@
 
 /obj/item/clothing/suit/hooded/wintercoat/captain
 	name = "captain's winter coat"
+	desc = "A special heavy winter jacket for a special person. Your actuely just wearing the carapace underneath. Not solving the nipple chaffing problem"
```
Comment (r1ks-iwnl) [inline code/modules/clothing/suits/miscellaneous.dm]:
```suggestion
	desc = "A special heavy winter jacket for a special person. You're actually just wearing the carapace underneath. Not solving the nipple chaffing problem"
```

**Comment by Archanial** [comment]
No, this is winter coat not body armor.

**Comment by bloons3** [comment]
Adding more pieces of armor to the Captain's locker is bad, especially for an item that is meant to be cosmetic

**Comment by ItsTisme** [comment]
> Adding more pieces of armor to the Captain's locker is bad, especially for an item that is meant to be cosmetic

I wanted  to make it cosmaticly more viable, since it provides very little protection against the carapce.

**Comment by PowerfulBacon** [comment]
It does just straight up say You're actually just wearing the carapace underneath. although it should have some downside considering it is cold proof and covers more limbs

**Comment by ItsTisme** [comment]
> It does just straight up say You're actually just wearing the carapace underneath. although it should have some downside considering it is cold proof and covers more limbs

Mainly the limb covering. Not sure what to do to debuff it though

**Comment by ItsTisme** [comment]
Maby have it have no fire resistance, since its winter coat and burning is the opposite of it

**Comment by ghost** [comment]
Buffing the style armors for the captain recreates the same problem that the HoS and Warden's lockers have: you can hand out loads of pieces of armor that are better than what even sec officers start with. If people want to take off their armor and walk around in puffy jackets or whatever else, they can deal with having less damage immunity. They don't NEED to be armored at all times

**Comment by ItsTisme** [comment]
> Buffing the style armors for the captain recreates the same problem that the HoS and Warden's lockers have: you can hand out loads of pieces of armor that are better than what even sec officers start with. If people want to take off their armor and walk around in puffy jackets or whatever else, they can deal with having less damage immunity. They don't NEED to be armored at all times

I dont see this happen ofton enough to see this as a problem. And captain is one of the few jobs that shoundlt have to compromise considering he holds most of the game chaning items

Not to 3 item objectevs are related to the captain(hand tele,anteque lazer, captain badge) and his AA is ininftly valued

**Comment by utilbird** [comment]
Captain has enough of an armoury in his locker as is.

**Comment by utilbird** [comment]
This has been stale for a month, closing to clear up some backlog.

**Pull request closed without merging**